src/features/DebugSession.ts

```diff
@@ -1,96 +1,96 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
-import vscode = require("vscode");
+import vscode from "vscode";
 import {
-    CancellationToken, DebugConfiguration, DebugConfigurationProvider,
-    ExtensionContext, WorkspaceFolder
+    CancellationToken,
```

Comment:
```suggestion
    vscode,
    CancellationToken,
```

Can these be combined now? Not sure if that works actually because TypeScript imports are weird.

Reply:
Let me update some imports but it should be doable, it wasn't before I enabled ESModuleInterop